### DISCUSSION TITLE
Has anyone upgraded to 7.0 and been using p2p volume streaming?

### DISCUSSION THREAD

**Discussion by evanchaoli** [concourse/concourse#6675] [General]
If yes, how much performance improvement you see from the feature?

**Comment by drahnr** [comment]
![image](https://user-images.githubusercontent.com/667047/112621479-178fc200-8e2a-11eb-88da-a6adb41a31fa.png)

I get a lot of those with two workers (containerized worker with houdini backend run with podman, and a containerd native) on the same host.

**Reply by kberzinch** [reply]
I get the same issue with just running the binary on an Amazon Linux 2 EC2 instance. I can't find anything particular in the worker logs that indicate a more specific reason, even with debug logs enabled.

**Reply by kberzinch** [reply]
This has become a fairly frequent occurrence. Does anyone have suggestions on how to troubleshoot this?

**Reply by kberzinch** [reply]
I found something in the logs after sending traces in honeycomb:

```json
{
  "timestamp": "2021-05-16T14:58:34.259993965Z",
  "level": "error",
  "source": "baggageclaim",
  "message": "baggageclaim.api.volume-server.stream-out.failed-to-stream-out",
  "data": {
    "error": "fork/exec /usr/bin/tar: cannot allocate memory",
    "session": "3.1.9860",
    "volume": "b6bec3cf-b975-425d-739e-5eaf72f3864a"
  }
}
```

This is on the source worker, which isn't shown in the UI.
